airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/job.py

```diff
@@ -138,13 +174,12 @@ def on_access_denied_job(self, **kwagrs) -> AirbyteTracedException:
     def on_job_with_errors(self, errors: List[Mapping[str, Any]]) -> AirbyteTracedException:
         raise ShopifyBulkExceptions.BulkJobUnknownError(f"Could not validate the status of the BULK Job `{self.job_id}`. Errors: {errors}.")
 
-    @limiter.balance_rate_limit(api_type=ApiTypeEnum.graphql.value)
     def job_check_for_errors(self, response: requests.Response) -> Iterable[Mapping[str, Any]]:
         try:
             return response.json().get("errors") or response.json().get("data", {}).get("bulkOperationRunQuery", {}).get("userErrors", [])
         except (Exception, JSONDecodeError) as e:
             raise ShopifyBulkExceptions.BulkJobBadResponse(
-                f"Couldn't check the `response` for `errors`, response: `{response.text}`. Trace: {repr(e)}."
+                f"Couldn't check the `response` for `errors`, status: {response.status_code}, response: `{response.text}`. Trace: {repr(e)}."
```

Comment:
Improved the logging by adding the response status code so we can observe it without the reproduction.